src/store/Store.js

```diff
@@ -138,6 +168,11 @@ export default class Store {
     this._event.remove();
     this._getter = getNull;
     this._responses = {};
+
+    typeof this._unsubscribeDevTools === 'function' &&
+      this._unsubscribeDevTools();
+    typeof this._devToolsExtension !== 'undefined' &&
```

Comment:
Related to https://github.com/HubSpot/general-store/pull/58/files#r153040699

@Friss I think flow might be able to deal with the the nullable type if you make the type check here more specific. Maybe `typeof this._devToolsExtension === 'object'`? 

If you don't have any luck there, we can leave it as is.

Reply:
Yeah still no luck I tried searching around for similar problems and everything kinda seemed to leave it at a dead end. :/

Reply:
That's fine this is also a pretty old version of flow 🤷‍♂️